etl/pipelines/openai_record_enrichment_pipeline.py

```diff
@@ -28,22 +28,22 @@ def __init__(self, openai_pipeline_config: OpenaiPipelineConfig) -> None:
     def __create_question(self, record_key: RecordKey) -> ModelQuestion:
         """Return a question for an OpenAI model."""
 
-        match self.__openai_pipeline_config.enrichment_type:
+        record_key_with_spaces = record_key.replace("_", " ")
```

Comment:
Redundant code again. Should have a common function that suggests why the conversion is being done / what the target format is.

RecordKeys.to_prompt_friendly(...)